pkg/cmd/admin/policy/reconcile_clusterrolebindings.go

```diff
@@ -0,0 +1,279 @@
+		if kapierrors.IsNotFound(err) {
+			changedRoleBindings = append(changedRoleBindings, expectedClusterRoleBinding)
+			continue
+		}
+		if err != nil {
+			return nil, err
+		}
+
+		if updatedClusterRoleBinding, needsUpdating := computeUpdatedBinding(expectedClusterRoleBinding, actualClusterRoleBinding, o.Union); needsUpdating {
+			changedRoleBindings = append(changedRoleBindings, updatedClusterRoleBinding)
+		}
+	}
+
+	return changedRoleBindings, nil
+}
+
+// ReplaceChangedRoleBindings will reconcile all the changed system role bindings back to the recommended bootstrap policy
+func (o *ReconcileClusterRoleBindingsOptions) ReplaceChangedRoleBindings(changedRoleBindings []*authorizationapi.ClusterRoleBinding) error {
+	for i := range changedRoleBindings {
+		roleBinding, err := o.RoleBindingClient.Get(changedRoleBindings[i].Name)
+		if err != nil && !kapierrors.IsNotFound(err) {
+			return err
+		}
+
+		if kapierrors.IsNotFound(err) {
+			createdRoleBinding, err := o.RoleBindingClient.Create(changedRoleBindings[i])
+			if err != nil {
+				return err
+			}
+
+			fmt.Fprintf(o.Out, "clusterrolebinding/%s\n", createdRoleBinding.Name)
+			continue
+		}
+
+		// RoleRef is immutable, to reset this, we have to delete/recreate
+		if !kapi.Semantic.DeepEqual(roleBinding.RoleRef, changedRoleBindings[i].RoleRef) {
+			roleBinding.RoleRef = changedRoleBindings[i].RoleRef
+			roleBinding.Subjects = changedRoleBindings[i].Subjects
+
+			// TODO: for extra credit, determine whether the right to delete/create this rolebinding for the current user came from this rolebinding before deleting it
+			err := o.RoleBindingClient.Delete(roleBinding.Name)
+			if err != nil {
+				return err
+			}
+			createdRoleBinding, err := o.RoleBindingClient.Create(changedRoleBindings[i])
+			if err != nil {
+				return err
+			}
+			fmt.Fprintf(o.Out, "clusterrolebinding/%s\n", createdRoleBinding.Name)
+			continue
+		}
+
+		roleBinding.Subjects = changedRoleBindings[i].Subjects
+		updatedRoleBinding, err := o.RoleBindingClient.Update(roleBinding)
+		if err != nil {
+			return err
+		}
+
+		fmt.Fprintf(o.Out, "clusterrolebinding/%s\n", updatedRoleBinding.Name)
+	}
+
+	return nil
+}
+
+func computeUpdatedBinding(expected *authorizationapi.ClusterRoleBinding, actual *authorizationapi.ClusterRoleBinding, union bool) (*authorizationapi.ClusterRoleBinding, bool) {
+	needsUpdating := false
+
+	// Always reset the roleref if it is different
+	if !kapi.Semantic.DeepEqual(expected.RoleRef, actual.RoleRef) {
+		needsUpdating = true
+	}
+
+	missingExpectedSubjects, extraActualSubjects := diff(expected.Subjects, actual.Subjects)
+	// Always add missing expected subjects
+	if len(missingExpectedSubjects) > 0 {
+		needsUpdating = true
+	}
+	// extra subjects only require a change if we're not unioning
+	if len(extraActualSubjects) > 0 && !union {
+		needsUpdating = true
+	}
+
+	if !needsUpdating {
+		return nil, false
+	}
+
+	var updated authorizationapi.ClusterRoleBinding = *expected
```

Comment:
same issue in reconcile-cluster-roles... should we stomp for now assuming the bootstrap ones don't set labels/annotations?